tests/bluetooth/tester/src/btp/bttester.h

```diff
@@ -43,6 +44,12 @@ uint8_t tester_init_mesh(void);
 uint8_t tester_unregister_mesh(void);
 void tester_handle_mesh(uint8_t opcode, uint8_t index, uint8_t *data, uint16_t len);
 
+uint8_t tester_init_vcp(void);
+uint8_t tester_unregister_vcp(void);
+void tester_handle_vcs(uint8_t opcode, uint8_t index, uint8_t *data, uint16_t len);
```

Comment:
If all of these shall follow the pattern of `(uint8_t opcode, uint8_t index, uint8_t *data, uint16_t len)`, should we `typedef` them? 

We could then consider putting these into a lookup table, rather than defining these functions each time like this. Could be something like the `struct event_handler normal_events` from `hci_core.c`

Reply:
I think this is also not the PR for that kind of change, I want to focus on adding those services, and then in the future we can perform that kind of refactors.

Reply:
Sure, we can do that. Perhaps create an GH enhancement issue for that?

Reply:
Once it is created, please add a reference to it here as well :D

Reply:
#53814 created